enigma-swing/src/main/java/cuchaz/enigma/gui/elements/MenuBar.java

```diff
@@ -399,6 +403,23 @@ private static void prepareOpenMenu(JMenu openMenu, Gui gui) {
 		}
 	}
 
+	private static void prepareMaxRecentFilesMenu(JMenu maxRecentFilesMenu) {
+		ButtonGroup group = new ButtonGroup();
+
+		for (int i = 5; i <= 30; i += 5) {
+			JRadioButtonMenuItem item = new JRadioButtonMenuItem(Integer.toString(i));
+			group.add(item);
+
+			if (UiConfig.getMaxRecentFiles() == i) {
+				item.setSelected(true);
+			}
+
+			int finalI = i;
+			item.addActionListener(event -> UiConfig.setMaxRecentFiles(finalI));
+			maxRecentFilesMenu.add(item);
+		}
```

Comment:
would it be wiser to use a numerical input or is that just not possible?

Reply:
honestly? for some reason I just defaulted to a little menu thing
that's absolutely possible if I pull up a dialog